browser/ui/BUILD.gn

```diff
@@ -106,6 +106,8 @@ source_set("ui") {
   # It doesn't make sense to view the webcompat webui on iOS & Android.
   if (!is_android && !is_ios) {
     sources += [
+      "ai_chat/tab_informer.cc",
+      "ai_chat/tab_informer.h",
```

Comment:
Adding these within _this if-statement_ gives the impression the Tab Informer work is somehow related to the WebCompat Reporter work.